snap/snapcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: zaproxy
+version: '2.7.0' 
+summary: OWASP ZAP, a tool for finding vulnerabilities in web applications
+description: |
+  The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular free security tools
+  and is actively maintained by hundreds of international volunteers.
+  It can help you automatically find security vulnerabilities in your web applications
+  while you are developing and testing your applications.
+  Its also a great tool for experienced pentesters to use for manual security testing.
+
+grade: devel
+confinement: devmode
+icon: /snap/zap256x256.png
+
+apps:
+  zaproxy:
+    command: desktop-launch $SNAP/wrapper
+
+parts:
+  zaproxy:
+    after: [desktop-glib-only]
+    source: https://github.com/zaproxy/zaproxy/releases/download/2.7.0/ZAP_2.7.0_Linux.tar.gz
+    plugin: dump
+    build: |
```

Comment:
The `parts` could be simplified if we use the [jdk plugin](https://github.com/snapcore/snapcraft/blob/master/snapcraft/plugins/jdk.py), e.g.:
```yaml
parts:
  java:
    plugin: jdk
    source: .
    build-packages:
      - ca-certificates-java
  zaproxy:
    after: [desktop-glib-only]
    plugin: dump
    source: https://github.com/zaproxy/zaproxy/releases/download/2.7.0/ZAP_2.7.0_Linux.tar.gz
```
we wouldn't need to specify additional packages and the `wrapper` would no longer be needed (the env variables are already set).